lib/karto_sdk/include/karto_sdk/Mapper.h

```diff
@@ -2401,6 +2408,8 @@ class KARTO_EXPORT Mapper : public Module
     ar & BOOST_SERIALIZATION_NVP(m_pMinimumAnglePenalty);
     ar & BOOST_SERIALIZATION_NVP(m_pMinimumDistancePenalty);
     ar & BOOST_SERIALIZATION_NVP(m_pUseResponseExpansion);
+    ar & BOOST_SERIALIZATION_NVP(m_pMinPassThrough);
```

Comment:
These 2 lines are what you'd need to remove for serialization

Reply:
Ok, nice, as I was thinking